NanoAnalysis/test/prod/samplesNano_2024_MC_ggH_AC.csv

```diff
@@ -0,0 +1,16 @@
+identifier,process,crossSection=-1,BR=1,execute=True,dataset,prefix,pattern,splitLevel,::variables,::pyFragments,comment
+ggH0PM_M125                      ,,52.23,0.0002745,,/GluGluH-Hto2Zto4L_Par-g1-1-M-125_TuneCP5_13p6TeV_powheg-jhugen-pythia8/RunIII2024Summer24NanoAODv15-150X_mcRun3_2024_realistic_v2-v2/NANOAODSIM,dbs,root://cms-xrd-global.cern.ch/,2,GENXSEC=578.006;GENBR=0.0002745;ADDLHEKINEMATICS=True;VVMODE=1;VVDECAYMODE=0;APPLY_QCD_GGF_UNCERT=True;LEPTON_SETUP=2024;ADD_ALLEVENTS=True;APPLYELECORR=True;APPLYMUCORR=True;APPLYJETCORR = True;,,
```

Comment:
The problem is somply that the variable list should not end with ";" :
`...APPLYJETCORR = True;,,` -> `...APPLYJETCORR = True,,`
Please also note that:
- you need too add NANOVERSION=15
- `APPLYELECORR=True;APPLYMUCORR=True;APPLYJETCORR=True` are all defaults, so I would suggest to remvove them to keep the file as readable as possible,